src/pybind/mgr/cephadm/tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+envlist = fix
+skipsdist = true
+
+[flake8]
+max-line-length = 100
+exclude =
+    .tox,
+    .vagrant,
+    __pycache__,
+    *.pyc,
+    templates,
+
+[autopep8]
+addopts =
+    --max-line-length {[flake8]max-line-length}
+    --exclude "{[flake8]exclude}"
+    --in-place
+    --recursive
+
+[testenv:fix]
+basepython = python3
+deps = -rrequirements-fix.txt
+commands = autopep8 {[autopep8]addopts} {posargs:.}
```

Comment:
OK, if this is your workflow to run `tox`from the mgr directory, then it will make sense. But i will only process the `cephadm` directory with `autopep8`, other modules need to add themselve to the tox.ini if they want to auto-format their code.